scss/_variables.scss

```diff
@@ -1 +1,44 @@
-$color-primary: #fff000;
+$color-primary: #FAFAFA;
+$grey:#3d4148;
+$container-width:1440px;
+%centering{
```

Comment:
In `variables` file should be placed only variables

Reply:
done